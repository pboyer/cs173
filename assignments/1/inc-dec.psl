defvar x = 5 in {
  print(x++);
  print(" ");
  print(++x);
  print(" ");
  print(x--);
  print(" ");
  (--x);
}