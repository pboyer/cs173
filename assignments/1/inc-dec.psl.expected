5 7 7 5
